github_app.go

```diff
@@ -205,11 +205,14 @@ func getInstallationIDFromCache(ctx context.Context, app int64, owner, repo stri
 	return installation.InstallationID, nil
 }
 
-type githubClient struct {
+// WrappedGitHubClient adds a Close method to the GitHub client that
+// puts the underlying HTTP client back into the pool.
+type WrappedGitHubClient struct {
```

Comment:
Would it be reasonable to just call this GithubClient instead of WrappedGithubClient?